packages/compass-components/src/constants/mongodb-ace-theme-query.ts

```diff
@@ -1,131 +0,0 @@
-import { palette } from '@leafygreen-ui/palette';
-
-const mongodbAceThemeQueryCssText = `
-.ace-mongodb-query .ace_scroller {
-line-height: 14px;
-background: none;
-}
-.ace-mongodb-query .ace_gutter {
-background: ${palette.white};
-color: ${palette.gray.base};
-}
-.ace-mongodb-query  {
-background: ${palette.white};
-color: ${palette.black};
-}
-.ace-mongodb-query .ace_placeholder {
-font-family: inherit;
-transform: none;
-opacity: 1;
-margin: 0;
-padding: 6px 9px !important;
-}
```

Comment:
This "query" theme was an exact copy of the other ace theme with this placeholder styles added. They seem relevant to both themes, so instead of keeping two themes, I merged them into one